lib/airbrake/rack/middleware.rb

```diff
@@ -40,6 +38,14 @@ def notify_airbrake(exception, env)
         notice = NoticeBuilder.new(env).build_notice(exception)
         Airbrake.notify(notice)
       end
+
+      # The internal framework middlewares store exceptions inside the Rack
+      # env. See: https://goo.gl/Kd694n
+      def framework_exception(env)
+        env['rack.exception'] ||
```

Comment:
Could you please move this to the end of the `||` pipeline, so it has least priority. As of now it seems like it's the default key that comes from Rack, which is not true. Could you please also explicitly mention that it's a Goliath thing (and link to it, too)?